tests/scenes/atmosphere/test_particle_layer.py

```diff
@@ -95,3 +98,45 @@ def test_particle_layer_radprops(mode_mono, test_dataset):
     assert all([coord in ds.coords for coord in expected_coords]) and all(
         [var in ds.data_vars for var in expected_data_vars]
     )
+
+
+@pytest.mark.parametrize("wavelength", np.linspace(500.0, 1500.0, 11) * ureg.nm)
+def test_particle_layer_eval_sigma_t(mode_mono, wavelength, test_dataset):
+    r"""Spectral dependance of extinction is accounted for.
+
+    If :math:`\sigma_t(\lambda)` denotes the extinction coefficient at the
+    wavelength :math:`\lambda`, then the optical thickness of a uniform
+    particle layer is :math:`\tau(\lambda) = \sigma_t(\lambda) \, \Delta z`
+    where :math:`\Delta z` is the layer's thickness.
+    It follows that:
+
+    .. math::
+
+       \frac{\tau(\lambda)}{\tau(550\, \mathrm{nm})} =
+       \frac{\sigma(\lambda)}{\sigma(550\, \mathrm{nm})}
+
+    which is what we assert in this test.
+    """
+    tau_550 = 1.0 * ureg.dimensionless
+    layer = ParticleLayer(
+        dataset=test_dataset,
+        bottom=0.0 * ureg.km,
+        top=1.0 * ureg.km,
+        distribution={"type": "uniform"},
+        n_layers=1,
+        tau_550=tau_550,
+    )
+
+    # layer optical thickness @ running wavelength
+    spectral_ctx = SpectralContext.new(wavelength=wavelength)
+    radprops = layer.eval_radprops(spectral_ctx)
```

Comment:
If you need only the extinction coefficient, you should rather use `eval_sigma_t()` (`eval_radprops()` is very wasteful).

Reply:
Indeed. I wanted to test `eval_radprops` instead of `eval_sigma_t` because I was looking for a mistake we might have made with how radiative properties are computed and `eval_radprops` is the method called by `AbstractHeterogeneousAtmosphere.kernel_media`. It looks like there is nothing wrong in how `eval_radprops` is using `eval_sigma_t` so we may just test `eval_sigma_t` instead, indeed.